docs/doc_v1.0/Build_documentation.md

```diff
@@ -121,20 +145,63 @@ go on in timesteps.
 Infected: the number of the population initially infected at t=0. Must
 be greater than 0.
 
-Seed: the seed for the random number generator. Will be modified by the
-command line argument if included. This ensure a reproducible output.
-
 Steady state: if there are still active infections when the contact data
 has been processed, the steady state flag enables the active cases to
 reach a resolution (recovered or dead) with no further random infections
 occurring.
 
-Contacts file: points to the CSV file being used in this run.
+(Optional) Seed: Can be specified on the command line or fixed.
 
 ![](image5.png)
 
 Figure 5. the run settings file
 
+
+## AgeData
+
+The age data file contains a list of id and age. This has been added 
+to allows the user to use age graduated contact files. Should the file 
+be longer than the population, additional input will be ignored. Should 
+it be shorter, the ages will be generated as per the population settings file. 
+
+![](ageData.png)
+
+## Isolation Policies
+
+The isolation policy can be split into 3 sections.
+
+### Virus Policy
+
+This policy determines the probability of a person in a given compartment 
+isolating, and for how long they will isolate. These can be modified and 
+varied using the distribution times. Also the time in isolation can vary 
+between ABSOLUTE or CONTACT_TIME. This means if a person becomes infected 
+they will either isolate for the isolation time from either the time of contact 
+or the time they are aware of being infected.
+![](virusPolicy.png)
+
+### Default Policy
+
+The default policy allows a global baseline to be set. For example, 
+in an older population 10% of the population may shield.
```

Comment:
We do not have the versatility yet to condition the policy on the individuals age, but definitely can be added in future with the current framework